syncd/syncd_flex_counter.cpp

```diff
@@ -185,11 +273,12 @@ void FlexCounter::addQueueCounterPlugin(
 }
 
 void FlexCounter::removeCounterPlugin(
-        _In_ std::string sha)
+        _In_ std::string sha,
+        _In_ uint32_t pollInterval)
 {
     SWSS_LOG_ENTER();
 
-    FlexCounter &fc = getInstance();
+    FlexCounter &fc = getInstance(pollInterval);
```

Comment:
>getInstance [](start = 22, length = 11)

remove instance?

Reply:
No need to remove instance here.
We need to remove it only if port counters, queue counters or attr maps are empty according to the original code.